lib/sources.nix

```diff
@@ -1,16 +1,33 @@
 # Functions for copying sources to the Nix store.
 { lib }:
 
+let
+  inherit (builtins)
+    hasContext
+    match
+    readDir
+    readFile
+    storeDir
+    tryEval
+    ;
+  inherit (lib)
+    filter
+    getAttr
+    isString
+    pathExists
+    split
+    ;
```

Comment:
import list correct

The first file in the commit moves everything from `builtins` to `lib`, see discussion above. We should be consistent.

Reply:
This only applies to `readFile`, which I've just done. I'd like to leave adding more builtins to lib out of scope for this PR.